Scripts/Proxies/TwoHandedProxyBase.cs

```diff
@@ -37,14 +36,14 @@ abstract class TwoHandedProxyBase : MonoBehaviour, IProxy, IFeedbackReceiver, IS
 
 		protected Transform m_LeftHand;
 		protected Transform m_RightHand;
-		readonly List<Material> m_Materials = new List<Material>();
 		readonly List<ProxyFeedbackRequest> m_FeedbackRequests = new List<ProxyFeedbackRequest>();
 
 		protected Dictionary<Node, Transform> m_RayOrigins;
 
-		List<Transform> m_ProxyMeshRoots = new List<Transform>();
+		bool m_Hidden;
 
-		readonly Dictionary<Node, Dictionary<VRInputDevice.VRControl, ProxyHelper.ButtonObject>> m_Buttons = new Dictionary<Node, Dictionary<VRInputDevice.VRControl, ProxyHelper.ButtonObject>>();
+		readonly Dictionary<Node, Dictionary<VRInputDevice.VRControl, List<ProxyHelper.ButtonObject>>> m_Buttons =
```

Comment:
Is this to handle button-combinations?

Reply:
This specifically came up because the Vive grip button is two physical buttons, which need separate renders and transforms, but are still considered one "input control."